conbench/templates/app.html

```diff
@@ -5,6 +5,7 @@
 <head>
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="robots" content="noindex">
```

Comment:
once again, this is a powerful technique documented here: https://developers.google.com/search/docs/crawling-indexing/block-indexing